### ISSUE TITLE
Bug: Contributor Insights pages all show bdougie and brandon

### ISSUE THREAD

**Issue by BekahHW** [open-sauced/app#4117]
### Describe the bug

![image](https://github.com/user-attachments/assets/acb7d546-32f9-4b48-9c78-2f4a3091c6f0)

### Steps to reproduce

1. Go to any contributor insights page (or use [this one](https://app.opensauced.pizza/workspaces/1b814876-2007-4125-91bf-9792b93ee6d2/contributor-insights/28c8cf6f-a117-4e35-b7a2-676cebe17e1a/overview))
2. Next to the title, see bdougie and brandon

**Comment by jpmcb** [comment]
Those are the members of the workspace who have access to that contributor insight. For example, in this contributor insight:

![Screenshot 2024-09-23 at 10 59 43 AM](https://github.com/user-attachments/assets/68819638-5855-4407-9fa3-fafbe68f8223)

I see bdougie, nick, and myself since those are the members of the workspace:

![Screenshot 2024-09-23 at 10 59 09 AM](https://github.com/user-attachments/assets/5d092fda-d7b9-474d-92d7-aff1e3242bad)

I agree it's really not obvious what those avatars are

**Comment by BekahHW** [comment]
@jpmcb 🤦‍♀️ I definitely didn't get that, especially with the pipe and the number of contributors next to it.

**Comment by nickytonline** [comment]
It probably would make sense to clarify that in the header or surface it somewhere else on that page. Thoughts @isabensusan?

**Comment by isabensusan** [comment]
Yeah, this is definitely confusing -- I'll assign this issue to myself and come up with a better solution!
